### PR TITLE
add start up func

### PR DESCRIPTION
**Pull request by Mikazakaryan**
start up func which is some default pathes yet

### CODE REVIEW

**Comment by Mikazakaryan** [comment]
issues #13 #16 #17 #18

File: main/main.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include "../shell.h"
 
 int main(int argc, char **argv) {
-
+
```
Comment (fxfactorial) [inline main/main.cpp]: Look at your commits before you commit, use `tig`

Comment (fxfactorial) [inline main/main.cpp]: I don't want to accept commits with whitespace changes.

File: shell.cpp [hunk 2]
```diff
@@ -6,14 +6,17 @@ using std::cout;
 using std::cin;
 using std::endl;
 using std::stringstream;
+using std::find;
 
 struct Path {
   vector<string> paths;
+
```
Comment (fxfactorial) [inline shell.cpp]: whitespace

File: shell.cpp [hunk 3]
```diff
@@ -36,8 +40,16 @@ struct Path {
       }
     }
   }
+
 };
 
+void start_up(Path & all_pathes){
```
Comment (fxfactorial) [inline shell.cpp]: start up is supposed to read from a file like: `~/.iteraterc` that will have shell commands that we read just like any other commands.

File: shell.cpp [hunk 4]
```diff
@@ -104,10 +113,34 @@ void fork_exec(vector<string> vector_arg){
 
 void get_comand(void) {
   string input_line;
+  static bool firs_call = true;
+  static Path all_pathes;
```
Comment (fxfactorial) [inline shell.cpp]: Why is this static

Comment (Mikazakaryan) [inline shell.cpp]: because it's have to be run once in program, not in every loop

Comment (fxfactorial) [inline shell.cpp]: Find another way, I don't like static

**Pull request closed without merging**